test-utils/build.gradle

```diff
@@ -6,6 +6,8 @@ android {
 dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
     implementation project(":common")
+    implementation project(":public")
```

Comment:
we needed these so that the stubs files could access `ProductType `and the product type conversions